### DISCUSSION TITLE
Limit relations by entry type

### DISCUSSION THREAD

**Discussion by RitterKnightCreative** [craftcms/cms#8524] [Ideas] [answered]
Would be great when an author is selecting related entries, Craft could limit by entry type.

Was building a site for a client and we needed to have a child/parent relationship between the top level entries and the children (but only the parent entries should be selected in the relations) and separate Category or Section didn't make sense.

Similar reading:
https://craftcms.stackexchange.com/questions/20368/limit-a-field-to-entries-by-entry-type

Related:
https://github.com/craftcms/cms/issues/853
https://github.com/craftcms/cms/issues/1628

**Comment by Anubarak** [comment]
Not really an answer but there is already a well working plugin that adds this field. Craft 2 [entriessubset](https://github.com/nfourtythree/craft-entriessubset) and for Craft 3 [craft3-entriessubset](https://github.com/nfourtythree/craft3-entriessubset)

**Comment by RitterKnightCreative** [comment]
Thanks Robin. As always, would be nice to see this built-into Craft but definitely will use this.

**Comment by brandonkelly** [comment]
Semi-related to #838, and will be possible to implement this yourself when we get around to #1630.

**Comment by brandonkelly** [comment, marked as answer]
We’ve added this for Craft 4! See #10393 for details.
